avm/res/dev-test-lab/lab/main.bicep

```
@@ -338,6 +341,19 @@ module lab_costs 'cost/main.bicep' = if (!empty(costs)) {
   }
 }
 
+module lab_secrets 'secret/main.bicep' = [
+  for (secret, index) in (secrets ?? []): {
+    name: '${uniqueString(deployment().name, location)}-Lab-Secrets-${index}'
+    params: {
+      labName: lab.name
+      name: secret.name
+      value: secret.value
+      enabledForArtifacts: secret.?enabledForArtifacts ?? false
+      enabledForVmCreation: secret.?enabledForVmCreation ?? false
```

```suggestion
      enabledForArtifacts: secret.?enabledForArtifacts
      enabledForVmCreation: secret.?enabledForVmCreation
```
Both are anyways the module's default and will be used automatically